extensions/sql-migration/src/wizard/loginSelectorPage.ts

```diff
@@ -44,6 +44,8 @@ export class LoginSelectorPage extends MigrationWizardPage {
 	private _systemLoginTable!: azdata.TableComponent;
 	private _systemLoginTableValues!: any[];
 	private _filterSystemLoginTableValue!: string;
+	private _successfullyValidatedLogins: Set<string> = new Set<string>();
```

Comment:
Why are we specifically using set here?

Reply:
This is because before proceeding to migration, we are checking whether all the selected logins are presented in this set of _successfullyValidatedLogins, and accessing in set is O(1)